sites/tvtv.us/tvtv.us.channels.xml

```diff
@@ -1575,6 +1575,7 @@
   <channel site="tvtv.us" lang="en" xmltv_id="TeleHit.mx" site_id="34394">TeleHit</channel>
   <channel site="tvtv.us" lang="en" xmltv_id="TelemundoEast.us" site_id="73245">Telemundo East (National Feed)</channel>
   <channel site="tvtv.us" lang="en" xmltv_id="TelemundoWest.us" site_id="17983">Telemundo West (National Feed)</channel>
+  <channel site="tvtv.us" lang="en" xmltv_id="TeleOnce.pr" site_id="17983">TeleOnce (San Juan, PR)</channel>

```

Comment:
this `site_id` is already used

Reply:
Corrected using 12569

https://tvtv.us/api/v1/lineup/USA-PR69606-X/grid/2023-11-04T04:00:00.000Z/2023-11-05T03:59:00.000Z/12569

![image](https://github.com/iptv-org/epg/assets/149446412/e9c03a2a-0c08-4f68-80b4-83b260e5039c)